index.js

```diff
@@ -3,30 +3,76 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+  let accumulator = false
+  if (rolls.length < 1){
+    accumulator = true
+  }
+  for (const roll of rolls){
+    if (typeof roll === 'number'){
+      accumulator = true
+    } else {
+      accumulator = false
+    }
+  }
+  return accumulator
+}
 
 /**
  * Finds a value in an array. If that value is in the array, returns it. Otherwise, returns `null`.
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @param {number} value - A specific value to find.
  * @returns {*} - The found value or `null`.
  */
-function findValue(rolls, value) {}
+function findValue(rolls, value) {
+  let accumulator = null
+
+  if (rolls.length === 0){
+    return accumulator
+  }
```

Comment:
This conditional is not necessary. If your array is empty, it will not enter the for loop, therefore hitting line 38 and returning null;